src/core/screen.js

```diff
@@ -68,6 +69,9 @@ export class Screen extends Phaser.Scene {
         this._data = data;
         this.cameras.main.scrollX = -CAMERA_X;
         this.cameras.main.scrollY = -CAMERA_Y;
+        this.pageIdx = -1;
+        this.pageItems = [];
+        data.config && this.events.once("create", nextPage(this));
```

Comment:
Yeah -
There's a bit of crossover between the background items work and the narrative screen.
If no pages are configured then it uses all the audio and all the tweens available as if they are page zero. 

In the above, pageItems ( - maybe there's a better name than pageItems?) is where currently running tweens and audio are stored so they can be cancelled so that needs to be shared.

pageIdx is narrative specific so maybe that should be moved to narrative.

Any thoughts on the above?

Reply:
ye pageItems kind of sounds like it includes background items too, apart from that sounds fine